src/components/molecules/ScheduleEvent/ScheduleEvent.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { MouseEventHandler, useCallback } from "react";
+import classNames from "classnames";
+import { useCss } from "react-use";
+
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faBookmark as solidBookmark } from "@fortawesome/free-solid-svg-icons";
+import { faBookmark as regularBookmark } from "@fortawesome/free-regular-svg-icons";
+
+import { SCHEDULE_HOUR_COLUMN_WIDTH_PX } from "settings";
+
+import { PersonalizedVenueEvent } from "types/venues";
+
+import { isEventLive } from "utils/event";
+
+import { ONE_HOUR_IN_MINUTES } from "utils/time";
+
+import {
+  addEventToPersonalizedSchedule,
+  removeEventFromPersonalizedSchedule,
+} from "api/profile";
+import { useUser } from "hooks/useUser";
+
+import { calcStartPosition } from "components/molecules/Schedule/Schedule.utils";
+
+import "./ScheduleEvent.scss";
+
+export interface ScheduleEventProps {
+  event: PersonalizedVenueEvent;
+  scheduleStartHour: number;
+  personalizedEvent?: boolean;
+}
+
+export const ScheduleEvent: React.FC<ScheduleEventProps> = ({
+  event,
+  scheduleStartHour,
+  personalizedEvent: isPersonaliezedEvent = false,
```

Comment:
[NB]
```suggestion
  personalizedEvent: isPersonalizedEvent = false,
```

Reply:
This would be good to do before this PR gets merged.

Reply:
fixed: https://github.com/sparkletown/sparkle/pull/1302/commits/bbb1862c171ac2cbb4b922d6b03f6a25b98dcc8f